book/intro.md

```diff
@@ -1,25 +1,19 @@
-1) Create open, executable, and web-hosted tutorials that encapsulate fundamental ocean acoustics knowledge and techniques as living documents, and
-2) Grow the ocean acoustics education and research community through education and collaborative resource deveopment workshops.
+The Bridge to Ocean Acoustics and Technology (BOAT) program aims to broaden access to ocean acoustics by fostering a vibrant, collaborative, and inclusive community. 
 
-In the current pilot stage, we are building tutorials on fundamental ocean acoustics concepts that can serve as "blueprints" for further developments on more advanced topics.
+Our focuses areL
+1) Developing open, executable, and web-hosted tutorials that encapsulate fundamental ocean acoustics knowledge and techniques as living documents.
+2) Growing the ocean acoustics education and research community through interactive and collaborative workshops.
 
-Our long-term goal is to build a strong learner-research-contributor community who jointly support this program to expand access to ocean acoustics theory and practice and to recruit the next generation of talent to this field.
-
-
-<!--📖 On this JupyterBook website you'll find [tutorials](tutorials/index). All tutorials are Jupyter Notebooks, 
-designed to be run interactively, but also rendered on this website for convenience.-->
-
-<!-- 👩‍💻 During a Hackweek teams work collaboratively on different projects. Read more about the projects and results on our [projects page](projects/index) -->
-
-<!-- 💡 Learn more about hackweeks hosted by the [University of Washington eScience Institute](https://uwhackweek.github.io/hackweeks-as-a-service/intro.html), or check out our publication describing the hackweek educational model {cite:p}`Huppenkothen2018`. -->
+In the current pilot stage, we are building foundational tutorials on core ocean acoustics concepts that can serve as "blueprints" for further developments on advanced topics and techniques. We will also host education workshops to engage learners from diverse backgrounds to "open up the funnel" for student intake into ocean acoustics.
 
 
 ## The BOAT Philosophy
 
-We build open, executable tutorials to connect learners to ocean acoustics knowledge on modern computing infrastructure.
+We build open, executable tutorials to connect learners to ocean acoustics knowledge on modern computing infrastructure. This approach fosters the development of physical and mathematical intuitions through hands-on computational examples first, inspiring and motivating learners to dive deeper into specific topics of their choice. 
+
+Looking ahead, we envision maintaining a collection of tutorial modules backed by a persistent JupyterHub, for the ocean acoustics community to learn an explore collaboratively beyond in-person or virtual workshops.
```

Comment:
Something does not flow in this sentence. How about:

Looking ahead, we envision maintaining a collection of tutorial modules backed by a persistent cloud-hosted computing environment, which would provide a space for the ocean acoustics community to learn and explore collaboratively beyond in-person or virtual workshops.

I also changed JupyterHub because we have not introduced it before (I think).